app/src/main/res/layout/topic_overview_fragment.xml

```diff
@@ -22,7 +22,10 @@
       <ImageView
         android:id="@+id/topic_thumbnail_image_view"
         android:layout_width="0dp"
-        android:layout_height="200dp"
+        android:layout_height="210dp"
```

Comment:
Please do confirm the specifications from Chantal.It is supposed to be multiples of 4 or 8

Reply:
Changed it to 202dp as mentioned in mock.